hydra/_internal/hydra.py

```diff
@@ -188,19 +189,22 @@ def show_cfg(
         )
         if package == "_global_":
             package = None
-        if package is not None:
+
+        if package is None:
+            ret = cfg
+        else:
             ret = OmegaConf.select(cfg, package)
             if ret is None:
                 sys.stderr.write(f"package '{package}' not found in config\n")
                 sys.exit(1)
-            else:
-                if isinstance(ret, Container):
-                    print(f"# @package {package}")
-                    sys.stdout.write(OmegaConf.to_yaml(ret))
-                else:
-                    print(ret)
-        else:
-            sys.stdout.write(OmegaConf.to_yaml(cfg))
+            if not isinstance(ret, Container):
+                print(ret)
+                return
+            print(f"# @package {package}")
```

Comment:
what is the motivation here? it does not seem much cleaner.

Reply:
In the previous commit I had a duplicated block that looked like this:
```python
if resolve:
    OmegaConf.resolve(...)
sys.stdout.write(OmegaConf.to_yaml(...))
```
I wanted to remove the duplication, making just one call to the `resolve`/`to_yaml` at the end of the function.

Reply:
can you use if-else instead of if return?

Reply:
Done in https://github.com/facebookresearch/hydra/pull/1616/commits/79b514cc54ba694f0a85b3b8ee8d39d8cfca10ee.